### PR TITLE
Added Japanese translation

### PR DESCRIPTION
**Pull request by matsune**
- [x] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

Added japanese translation for all strings and rtf files.

### CODE REVIEW

File: iina/ja.lproj/KeyBinding.strings [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+"opt.fullscreen" = "フルスクリーン";
+"opt.ontop" = "最前面に固定";
+
+"opt.chapter" = "チャプタ";
+
+
+"read.set" = "{{property}}に{{value}}を設定";
+"read.add" = "{{property}} {{value}}";
+"read.multiply" = "{{property}}を{{factor}}倍";
+"read.cycle" = "{{#dir_down}}逆に{{/dir_down}} {{property}} オン/オフ";
```
Comment (lhc70000) [inline iina/ja.lproj/KeyBinding.strings]: "Cycle" doesn't only mean "on/off", e.g. cycle in subtitle tracks.

Also, I think it's better to remove the space after "逆に".

File: iina/ja.lproj/KeyBinding.strings [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+"iina.find-online-subs" = "オンラインで字幕を探す";
+"iina.save-downloaded-sub" = "ダウンロードした字幕を保存";
+
+"yes" = "オン";
+"no" = "オフ";
+"to" = "を";
+"by" = "x";
+"in" = "次の値のうち";
+"add" = "追加";
+"minus" = "削除";
```
Comment (lhc70000) [inline iina/ja.lproj/KeyBinding.strings]: "add" and "minus" here means "増加" and "減少".

I should change them to "increase" and "decrease"……

File: iina/ja.lproj/Contribution.rtf [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+\f2 \cf0 \expnd0\expndtw0\kerning0
+\'82\'b1\'82\'cc\'83\'76\'83\'8d\'83\'4f\'83\'89\'83\'80\'82\'cd\'83\'74\'83\'8a\'81\'5b\'83\'5c\'83\'74\'83\'67\'83\'45\'83\'46\'83\'41\'82\'c5\'82\'b7\'81\'42\'82\'a0\'82\'c8\'82\'bd\'82\'cd\
+\'82\'b1\'82\'ea\'82\'f0\'81\'41\'83\'74\'83\'8a\'81\'5b\'83\'5c\'83\'74\'83\'67\'83\'45\'83\'46\'83\'41\'8d\'e0\'92\'63\'82\'c9\'82\'e6\'82\'c1\'82\'c4\'94\'ad\'8d\'73\'82\'b3\'82\'ea\'82\'bd\
+
+\f0 \kerning1\expnd0\expndtw0 GNU
+\f2 \expnd0\expndtw0\kerning0
+\'88\'ea\'94\'ca\'8c\'f6\'8f\'4f\'97\'98\'97\'70\'8b\'96\'91\'f8\'8c\'5f\'96\'f1\'8f\'91
+\f3 (
+\f2 \'83\'6f\'81\'5b\'83\'57\'83\'87\'83\'93
+\f3 2
```
Comment (lhc70000) [inline iina/ja.lproj/Contribution.rtf]: This translation mentions version 2 (should be 3)?

Comment (matsune) [inline iina/ja.lproj/Contribution.rtf]: Oh sorry, this should be 3.

**Pull request merged**